content/en/docs/contribute/style/style-guide.md

```diff
@@ -115,45 +121,54 @@ represents.
-  <tr><td>Use single backticks to enclose inline code. For example, `var example = true`.</td><td>Use two asterisks (**) or an underscore (_) to enclose inline code. For example, **var example = true**.</td></tr><tr><td>Use triple backticks before and after a multi-line block of code for fenced code blocks.</td><td>Use multi-line blocks of code to create diagrams, flowcharts, or other illustrations.</td></tr><tr><td>Use meaningful variable names that have a context.</td><td>Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.</td></tr><tr><td>Remove trailing spaces in the code.</td><td>Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Use code style for inline code and commands" >}}
+Do | Don't
+:--| :-----
+The <code>kubectl run</code> command creates a Deployment. | The "kubectl run" command creates a Deployment.
+For declarative management, use <code>kubectl apply</code>. | For declarative management, use "kubectl apply".
+Enclose code samples with triple backticks. <code>(```)</code> | Enclose code samples with any other syntax.
+Use single backticks to enclose inline code. For example, `var example = true`. | Use two asterisks (**) or an underscore (_) to enclose inline code. For example, **var example = true**.
+Use triple backticks before and after a multi-line block of code for fenced code blocks. | se multi-line blocks of code to create diagrams, flowcharts, or other illustrations.
+Use meaningful variable names that have a context. | Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.
+Remove trailing spaces in the code. | Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well. 
+{{< /table >}}
 
 {{< note >}} 
 The website supports syntax highlighting for code samples, but specifying a language is optional. Syntax highlighting in the code block should conform to the [contrast guidelines.](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=141%2C143#contrast-minimum)
 {{< /note >}}
 
 ### Use code style for object field names
 
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>Set the value of the <code>replicas</code> field in the configuration file.</td><td>Set the value of the "replicas" field in the configuration file.</td></tr>
-  <tr><td>The value of the <code>exec</code> field is an ExecAction object.</td><td>The value of the "exec" field is an ExecAction object.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Use code style for object field names" >}}
+Do | Don't
+:--| :-----
+Set the value of the <code>replicas</code> field in the configuration file. | Set the value of the "replicas" field in the configuration file.
```

Comment:
```suggestion
Set the value of the <code>replicas</code> field in the configuration file. | Set the value of the `replicas` field in the configuration file.
```

I'd use backticks here too.

Reply:
Please replace the \<code> tags with backticks. So here it would be \`replicas\`  which then renders as `replicas`. 

@sftim - I think the part in the "Don't" column is correct -- replicas in quotes  "replicas"
MD code:
```
Do | Don't
:--| :-----
Set the value of the `replicas` field in the configuration file. | Set the value of the "replicas" field in the configuration file.
```
Renders as:
Set the value of the `replicas` field in the configuration file. | Set the value of the "replicas" field in the configuration file.

(I have to have the netlify rendered version up as I'm reviewing this)

Reply:
Replaced <code> with backticks.